gpjax/kernels.py

```diff
@@ -691,15 +691,15 @@ def __init__(
     ) -> None:
         super().__init__(compute_engine, active_dims, stationary, spectral, name)
 
-    def __call__(self, params: dict, x: jnp.DeviceArray, y: jnp.DeviceArray) -> Array:
+    def __call__(self, params: dict, x: jax.Array, y: jax.Array) -> Array:
         """Evaluate the kernel on a pair of inputs :math:`(x, y)` with length-scale parameter :math:`\ell`, :math:`\sigma` and power :math:`\kappa`.
 
         .. math::
             k(x, y) = \\sigma^2 \\exp \\Bigg( - \\Big( \\frac{\\lVert x - y \\rVert^2}{\\ell^2} \\Big)^\\kappa \\Bigg)
```

Comment:
We have quite a few `jax.Arrays` in `kernels.py` being using in the input typing and for the arg docstrings. For typing I would prefer `Float[Array, "shape"]`.